app/views/flush.py

```diff
@@ -0,0 +1,66 @@
+from app.globals import get_answer_store, get_metadata, get_questionnaire_store
+from app.submitter.submitter import SubmitterFactory
+from app.submitter.converter import convert_answers
+from app.utilities.schema import get_schema
+
+from app.authentication.permission_denied_exception import PermissionDeniedException
+from app.authentication.survey_data_exception import SurveyDataException
+from app.authentication.no_token_exception import NoTokenException
+from app.authentication.user import User
+from app.authentication.user_id_generator import UserIDGenerator
+from app.authentication.authenticator import jwt_decrypt
+
+from app.questionnaire.path_finder import PathFinder
+
+from flask import Blueprint, Response, request, session
+
+flush_blueprint = Blueprint('flush', __name__)
+
+
+@flush_blueprint.route('/flush', methods=['GET'])
+def flush_data():
+
+    if session:
+        session.clear()
+
+    encrypted_token = request.args.get('token')
+
+    if encrypted_token is None:
+        raise NoTokenException("Please provide a token")
+
+    decrypted_token = jwt_decrypt(encrypted_token)
+
+    roles = decrypted_token.get('roles')
+
+    if roles:
+        if 'flusher' in roles:
+            user = _get_user(decrypted_token)
+            _submit_data(user)
+            get_questionnaire_store(user.user_id, user.user_ik).delete()
+        else:
+            raise PermissionDeniedException("Your role doesn't have permission to flush data")
```

Comment:
Do we want to raise this exception, it will cause a web page to be returned which isn't ideal for an automated system. Why not just return Response(status=403)